Dockerfile

```diff
@@ -1,5 +1,5 @@
 # stage 1 Generate celestia-appd Binary
-FROM docker.io/golang:1.19.7-alpine3.17 as builder
+FROM docker.io/golang:1.20.3-alpine3.17 as builder
```

Comment:
It doesn't seem necessary to merge this dependabot PR until we update this repo to use Go 1.20